common/src/main/java/org/broadleafcommerce/common/web/BroadleafRequestProcessor.java

```diff
@@ -168,8 +168,8 @@ public void process(WebRequest request) {
                     if (isSandboxIdValid(sandboxId)) {
                         String queryString = "?" + SANDBOX_ID_PARAM + "=" + sandboxId;
                         url.append(queryString);
-                        if(hsr.getParameter("blIncludeMyChanges")!=null){
-                            url.append("&blIncludeMyChanges="+hsr.getParameter("blIncludeMyChanges"));
+                        if(hsr.getParameter(INCLUDE_MY_CHANGES_VAR) != null){
+                            url.append("&blIncludeMyChanges="+hsr.getParameter(INCLUDE_MY_CHANGES_VAR));
```

Comment:
here. instead of &blIncludeMyChanges=
reuse 
"&"+BroadleafIncludeMyChangesResolver.INCLUDE_MY_CHANGES_VAR+"="